app/src/main/java/com/vero/tasky/agenda/data/remote/network/dto/EventDto.kt

```diff
@@ -0,0 +1,17 @@
+package com.vero.tasky.agenda.data.remote.network.dto
+
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class EventDto(
+    val id: String,
+    val title: String,
+    val description: String,
+    val from: Long,
+    val to: Long,
+    val remindAt: Long,
+    val host: String,
+    val isUserEventCreator: Boolean,
+    val attendees: List<AttendeeDto>,
+    val photos: List<PhotoDto>
```

Comment:
For photos and attendees you'll need to create DB relations

You have 2 options here
1- Do the relationship
2- (easier) Just save each as json string. It doesnt scale in the long run, but its easier and the odds of you changing it are low